src/3d/CMakeLists.txt

```diff
@@ -122,6 +126,8 @@ SET(QGIS_3D_HDRS
   symbols/qgsabstract3dsymbol.h
   symbols/qgsline3dsymbol.h
   symbols/qgsline3dsymbol_p.h
+  symbols/qgslinematerial_p.h
+  symbols/qgslinevertexdata_p.h
```

Comment:
I think private headers shouldn't be installed normally, so no reason to list them here

Reply:
There are few cases when private include files may be transitively #included by some public headers so for now it seemed better to install these as well. Once we will declare stable api for the 3d library, we can clean that up (what is public and what is private may still change)...

Reply:
> There are few cases when private include files may be transitively #included by some public header

But that's not the case here, is it?

Reply:
Nope, just being consistent by adding them so we do not wonder later why some private headers are installed and some aren't :)